### ISSUE TITLE
 How long have you learned?

### ISSUE THREAD

**Issue by ShunIto2015** [dougsm/ggcnn#1]
Hello. I'd like to get the same results as your paper data.
Would you tell me time for learning(and GPU Spec) and final score?
Thank you.

**Comment by dougsm** [comment]
Hello,
Training usually only takes about 20-30 epochs (only about 1 or 2 hours on a GTX1070).  Final evaluation against the validation set is usually about 80%.

**Comment by ShunIto2015** [comment]
Thank you very mush!
